sql/src/main/java/io/crate/execution/dsl/phases/FileUriCollectPhase.java

```diff
@@ -92,6 +96,11 @@ public String compression() {
         return compression;
     }
 
+    @Nullable
```

Comment:
Would remove the Nullable here to avoid dealing with nulls, InputFormat should always be defined.

Reply:
ce7b4c77